_data/contributors.yml

```
@@ -22,3 +22,6 @@ Matthieu Solente:
 Niven Ranchhod:
   name: Niven Ranchhod
   url: https://bare-bones.dev/
+Bart en Hertog:
```

```suggestion
Bart den Hertog:
```
Looks like a little typo here, otherwise all good

---

(you can just click that 'commit suggestion' button if you are happy with the update)